Updated style.css (CSS Enhancements Issue)

Kallind opened the pull request:
added hover features, button features and changed colors to make it more catchy

Kallind committed:
Update style.css

added hover features, button features and changed colors to make it more catchy

Dv04 reviewed (changes requested):
Hey @Kallind, Your work looks great but requires some minor changes.

Before:
<img width="1440" alt="Screenshot 2023-10-11 at 9 29 30 PM" src="https://github.com/Dv04/Text_Detector/assets/62090715/c39b15c3-cbcf-461f-90e6-8422e4e654b2">

After:
<img width="1440" alt="Screenshot 2023-10-11 at 9 27 54 PM" src="https://github.com/Dv04/Text_Detector/assets/62090715/9e837b25-8421-46d8-bb97-86c4ba9a3a1b">

Changes Required:
- The hover effects were perfect but the image placeholder area should not have been removed.
- The colours should remain of one base. optimally the original one or the gray/blue style.

Kallind committed:
Revert "Update style.css"

This reverts commit 67383c1b3884c1f3b2095793526b1f8633317572.

Kallind committed:
Update style.css

changed the colour scheme back to orginal, added hover features on buttons and box shadow on the image. also added the image placeholder area back

Kallind commented:
changed the colour scheme back to orginal, added hover features on buttons and box shadow on the image. also added the image placeholder area back
let me know if you need any other changes, i hope it looks good now

Dv04 reviewed (approved):
Looks good.
